html/index.html

```diff
@@ -0,0 +1,48 @@
+<!Doctype html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Blog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="../css/base.css" rel="stylesheet">
+    <link href="../css/layout.css" rel="stylesheet">
+    <link href="../css/module.css" rel="stylesheet">
+    <link href="../css/theme.css" rel="stylesheet">
+    <link href="../css/state.css" rel="stylesheet">
+  </head>
+
+  <body>
+    <!-- //navigation -->
+    <header class="site-header clearfix">
+      <section class="heading group">
+        <h1>Portfolio</h1>
+        <h2>Something goes here1</h2>
+        <button class="hamburger" name="button"></button>
+    </header>
+
+    <main>
+      <div>
+        <h1>WENJING QIANG<h1>
+        <img src="../vendor/img1.png">
+      </div>
+      <ul>
+        <div ><li><a href="about.html">About</a></li></div>
```

Comment:
@cacizi41 You may have changed this since this pr, but we should not be linking to other pages here, and instead be revealing the sections within one page.